tests/ext/test_conversationhandler.py

```diff
@@ -289,6 +291,29 @@ def test_init_persistent_no_name(self):
                 self.entry_points, states=self.states, fallbacks=[], persistent=True
             )
 
+    def test_repr(self):
+        # This function tests behaviour inherited from BaseHandler.
+        # TODO should this test be repeated in all other subclasses of BaseHandler except for
+        #  ConversationHandler that has its own __repr__?
```

Comment:
IMO it would suffice to test this once in `test_basehandler`.

Reply:
Done

Reply:
Sorry, CH should then ofc be tested individually. But I see no need to test in all handler classes that simply inherit from BaseHandler :)

Reply:
OK, I think the test for CH is already in place :)